README.md

```diff
@@ -11,7 +11,11 @@ An exporter for [Amazon Cloudwatch](http://aws.amazon.com/cloudwatch/), for Prom
 
 ## Credentials and permissions
 
-The Cloudwatch Exporter uses the [AWS Java SDK](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/welcome.html), which offers [a variety of ways to provide credentials](http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/credentials.html). This includes the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` enviroment variables
+The Cloudwatch Exporter uses the
```

Comment:
That affects other things too, we've other exporters that have non-obvious capitalisation to those not used to them such as HAProxy.